src/backend/tests/work-packages.test.ts

```diff
@@ -59,6 +61,7 @@ describe('Work Packages', () => {
     jest.spyOn(changeRequestUtils, 'validateChangeRequestAccepted').mockImplementation(async (_crId) => {
       return prismaChangeRequest1;
     });
+    jest.spyOn(workPackageTransformer, 'default').mockReturnValue(sharedWorkPackage);
```

Comment:
What's this line for? Not part of your tests

Reply:
Its for my working test, this line is mocking basically what it returns by mocking the transformer that is called within the function I am testing. The reason it is here and not in my actual test is because thats what Anthony wanted.

Reply:
ye we want this mocked always tbh